### ISSUE TITLE
payload-file-path broken

### ISSUE THREAD

**Issue by rafi-fisker** [slackapi/slack-github-action#52]
### Description

This example does not work
![image](https://user-images.githubusercontent.com/72412693/151201195-afe75886-4a6b-4df1-bb41-9d0485e8d80b.png)

My action 
```
      - name: Notify success
        if: ${{ success() }}
        uses: slackapi/slack-github-action@v1.17.0
        with:
          payload-file-path: "./.github/workflows/payload-success.json"
```
My webhook url is set at the `job` level
```
jobs:
  tests:
    runs-on: ubuntu-latest
    env:
      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
``` 
I see this in logs
```
Warning: Unexpected input(s) 'payload-file-path', valid inputs are ['channel-id', 'slack-message', 'payload']
```
`payload-file-path` appears to be missing as an input here:
https://github.com/slackapi/slack-github-action/blob/main/action.yml

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [x] bug
- [ ] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] example code related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

Filling out the following details about bugs will help us solve your issue sooner.

#### Reproducible in:

package version: 1.17.0

node version:

OS version(s):

#### Steps to reproduce:

1. Create action with `payload-file-path` per example
2.
3.

#### Expected result:

Payload is used

#### Actual result:

Fails to send

#### Attachments:

Logs, screenshots, screencast, sample project, funny gif, etc.

**Comment by rafi-fisker** [comment]
You can see same behavior in your integration test https://github.com/slackapi/slack-github-action/runs/4789855821?check_suite_focus=true

**Comment by stevengill** [comment]
Nice catch @rafi-fisker!

I just pushed a fix up to the main branch

**Comment by stevengill** [comment]
Released version v1.18.0 https://github.com/slackapi/slack-github-action/releases/tag/v1.18.0
